src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ExpressionHelper.cs

```diff
@@ -19,13 +19,16 @@ public static string GetExpressionText(string expression)
             return string.Equals(expression, "model", StringComparison.OrdinalIgnoreCase) ? string.Empty : expression;
         }
 
-        public static string GetExpressionText(LambdaExpression expression)
+        public static bool GetExpressionText(LambdaExpression expression, out string expressionText)
```

Comment:
API change makes it look like a `TryGetBlah()` method, which this isn't. Suggest instead passing the `ExpressionTextCache` _in_ and keeping the `isMemberAccessExpressionOnly` details private.